website/_enemies/forest/quill.md

```diff
@@ -11,4 +11,6 @@ sprite_folder_name: porcupine
 sprite_image_name: spike
 ---
 
-Once redirected by the player with an attack, it can hit other enemies for 100 (!) damage, as demonstrated in top [Wild Den](#maps-level-wild-den) replays.
+Can be hit and redirected by entities on the opposing "team" to the previous owner. The entity that hits it becomes the new owner. Once redirected by the player with an attack, it can hit other enemies for 100 (!) damage, as demonstrated in top [Wild Den](#maps-level-wild-den) replays.
```

Comment:
This could probably be a bit more concise since I guess it doesn't actually need to be redirected before making the damage.

```suggestion
Can be hit and redirected by entities on the opposing "team" to the previous owner. The entity that hits it becomes the new owner.

It will hit enemies for 100 (!) damage, as demonstrated in top [Wild Den](#maps-level-wild-den) replays.
```

Reply:
I think it's important to say that it's redirected, because if the quill is not redirected by the player, it will not interact with other enemies.